osx/FontForge.app/Contents/Resources/breakpad/Makefile.am

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2015 by Ben Martin
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+#
+# Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# The name of the author may not be used to endorse or promote products
+# derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR IMPLIED
+# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO
+# EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+include $(top_srcdir)/mk/layout.am
+
+SUBDIRS = webinterface
+
+FFBP_FILES = localhost-8000.webloc
+FFAPP_FILES = minidump_stackwalk
+EXTRA_DIST = $(FFBP_FILES) $(FFAPP_FILES)
+
+if PLATFORM_OSX
+   ffbpdir = $(pkgdatadir)/osx/FontForge.app/Contents/Resources/breakpad
+   ffbp_DATA = $(FFBP_FILES)
+
+   ffappdir = $(pkgdatadir)/osx/FontForge.app/Contents/Resources/breakpad
+   ffapp_SCRIPTS = $(FFAPP_FILES)
+endif
+
+-include $(top_srcdir)/git.mk
```

Comment:
This is a 'try to include but don't complain' version of include;
http://www.jfranken.de/homepages/johannes/vortraege/make_inhalt.en.html#ToC7

It's used throughout, for example the Makefile.am in the root of fontforge:
https://github.com/fontforge/fontforge/blame/master/Makefile.am#L147

I'm not sure if git.mk is possibly unavailable in some cases. git.mk is not in extra_dist for the root directory so it is probably not in make dist; Maybe it makes more sense to have it in make dist and assume we can always find it. Unless the content in it clashes when you don't have a .git directory.